modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/treeview/NodeList.js

```diff
@@ -47,6 +48,12 @@ export default function NodeList({
 					onFocus(event);
 				}
 			}}
+			onMouseDown={(event) => {
+				if (onMouseDown) {
+					onMouseDown(event);
+					event.preventDefault();
```

Comment:
Can we put `event.preventDefault()` to `handleMouseDown` as well? It is also a part of higher-level component logic.

Reply:
Yes sure, changes pushed. 

Regards.

Reply:
@georgel-pop-lr Oh I'm sorry, I misspoke, I meant to **move** `event.preventDefault` from here to `handleMouseDown` :sweat_smile: . Can you please delete it here. I promise, this is the last change!

Reply:
@markocikos yes it seems strange to me that we needed in two places, change pushed :)

Reply:
Thanks for the quick updates! Pushing!